templates/terraform/google-sa/main.tf

```diff
@@ -3,15 +3,18 @@ resource google_service_account sa {
   provider = google.target
   account_id = var.account_id
   display_name = var.display_name
+  depends_on = [var.dependencies]
```

Comment:
Need to put the `[ ]` brackets here otherwise terraform yells that you need a static list here.